plugins/modules/system/open_iscsi.py

```diff
@@ -102,10 +107,18 @@
   community.general.open_iscsi:
     login: no
     target: iqn.1986-03.com.sun:02:f8c1f9e0-c3ec-ec84-c9c9-8bfb0cd5de3d
+
+- name: Override and disable automatic portal login on specific portal
+  community.general.open_iscsi:
+    login: no
+    portal: 10.1.1.250
+    auto_portal_startup: no
```

Comment:
```suggestion
    login: false
    portal: 10.1.1.250
    auto_portal_startup: false
```
Sorry I didn't spot this earlier, YAML booleans should always be `true` or `false` in the docs.